Segments/Kesmai.mapproj

```diff
@@ -145233,18 +145233,18 @@ else
         
         IsTethered = true,
     };
-    dragon.AddStatus(new NightVisionStatus(dragon));
+    demon.AddStatus(new NightVisionStatus(dragon));
```

Comment:
You need to change the new nvstatus(dragon) to new nvstatus(demon), otherwise compiler issues.